AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginIntegrationTests/DataStoreLocalUsageTests.swift

```diff
@@ -0,0 +1,289 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import XCTest
+
+import AmplifyPlugins
+import AWSMobileClient
+import AWSPluginsCore
+
+@testable import Amplify
+@testable import AmplifyTestCommon
+@testable import AWSDataStoreCategoryPlugin
+
+class DataStoreLocalUsageTests: LocalUsageIntegrationTestBase {
```

Comment:
suggestion: `DataStoreLocalStoreIntegrationTests` and `LocalStoreIntegrationTestBase`, we often refer to local store/remote store

Reply:
Renamed